_sass/_components/layout.scss

```diff
@@ -139,3 +143,17 @@
   @include margin($section-margins auto);
   max-width: 61rem;
 }
+
+.usa-content {
```

Comment:
Bc we have the variables import problem, it's not taking our new `max-width: $text-max-width;`. This will be removed when the variable problem is fixed.